crates/orderbook/openapi.yml

```
@@ -902,7 +906,9 @@ components:
           allOf:
             - $ref: "#/components/schemas/BigUint"
         executedFeeAmount:
-          description: The total amount of fees that have been executed for this order.
+          description: The total amount of the user signed `fee` that have been
```

to make it even clearer
```suggestion
          description: [DEPRECATED] The total amount of the user signed `fee` that have been
```